docs/src/preprocess/index.js

```diff
@@ -168,19 +167,10 @@ function extractCodeSnippet(filePath, identifier) {
   // We have our code snippet!
   let codeSnippet = lines.join("\n");
 
-  let startCharIndex = startMatch.index;
-  let endCharIndex = endMatch.index;
-
-  const startLine = getLineNumberFromIndex(codeSnippet, startCharIndex) + 1;
-  const endLine =
-    getLineNumberFromIndex(codeSnippet, endCharIndex) -
-    1 -
-    linesToRemove.length;
-
```

Comment:
This was incorrect. I've removed it. We already had startLineNum and endLineNum in this function, which give the right values.